cli/src/test/java/hudson/cli/CLIConnectionFactoryTest.java

```diff
@@ -0,0 +1,26 @@
+package hudson.cli;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CLIConnectionFactoryTest {
+
+    CLIConnectionFactory sut;
+
+    @BeforeEach
+    void setUp() {
+        sut = new CLIConnectionFactory();
+    }
+
+    @Test
+    void testBearerFromToken() {
+        Assert.assertEquals("Bearer some-token", sut.of("some-token").authorization);
+    }
+
```

Comment:
Would it be better if we also include  an Empty Bearer Token test ?

Reply:
I can add the test, but the cli will behave in the same was as if empty auth is passed to basicAuth, semantics are the same. What do you suggest?

Reply:
Okay, I do not have a deep understanding of basicAuth here, if I understand correctly if empty token is passed it will be handled normally and therefore we do not need to test it. Might need some additional review
CC @daniel-beck